R/zchunk_L201.en_nonco2.R

```diff
@@ -0,0 +1,358 @@
+                     "energy/A_regions",
+                     "temp-data-inject/L112.ghg_tg_R_en_S_F_Yh",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.en_ghg_emissions
+
+    L201.en_bcoc_emissions %>%
+      add_title("BC/OC emissions factors for energy technologies in all regions") %>%
+      add_units("Tg/EJ") %>%
+      add_comments("Take BC/OC emissions factors for energy technologies, filter to out_resources") %>%
+      add_comments("in model base years, rename to regional SO2, filter to district heat,") %>%
+      add_comments("and delete sectors with zero heating/cooling degree days.") %>%
+      add_legacy_name("L201.en_bcoc_emissions") %>%
+      add_precursors("common/GCAM_region_names",
+                     "energy/A_regions",
+                     "L114.bcoc_tgej_R_en_S_F_2000",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.en_bcoc_emissions
+
+    L201.nonghg_max_reduction %>%
+      rename(max.reduction = max_reduction) %>% # no idea why old data system renamed this
+      add_title("Maximum reduction for energy technologies in all regions") %>%
+      add_units("%") %>%
+      add_comments("comments describing how data generated") %>%
+      add_comments("can be multiple lines") %>%
+      add_legacy_name("L201.nonghg_max_reduction") %>%
+      add_precursors("common/GCAM_region_names",
+                     "emissions/A_regions", "energy/A_regions",
+                     "L151.nonghg_ctrl_R_en_S_T",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.nonghg_max_reduction
+
+    L201.nonghg_steepness %>%
+      add_title("Steepness of reduction for energy technologies in all regions") %>%
+      add_units("%") %>%
+      add_comments("comments describing how data generated") %>%
+      add_comments("can be multiple lines") %>%
+      add_legacy_name("L201.nonghg_steepness") %>%
+      add_precursors("common/GCAM_region_names",
+                     "emissions/A_regions", "energy/A_regions",
+                     "L151.nonghg_ctrl_R_en_S_T",
+                     "emissions/A51.steepness",
+                     "temp-data-inject/L244.DeleteThermalService") ->
+      L201.nonghg_steepness
+
+    L201.nonghg_max_reduction_res %>%
+      rename(max.reduction = max_reduction) %>% # no idea why old data system renamed this
+      add_title("Maximum reduction for resources in all regions") %>%
+      add_units("%") %>%
+      add_comments("comments describing how data generated") %>%
+      add_comments("can be multiple lines") %>%
+      add_legacy_name("L201.nonghg_max_reduction_res") %>%
+      same_precursors_as(L201.nonghg_max_reduction) ->
+      L201.nonghg_max_reduction_res
+
+    L201.nonghg_steepness_res %>%
+      add_title("Steepness of reduction for resources in all regions") %>%
+      add_units("%") %>%
+      add_comments("comments describing how data generated") %>%
```

Comment:
Following from above:

"The steepness is from an assumptions file (A_***). It was chosen to replicate the pollutant reduction rates for SO2 in GCAM3. Steepness is removed for technologies with maximum reduction rates of 0."

Again, very little happens in this file other than to map to regions and choose the right column names.